packages/letters-of-credit/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "argentina-app",
-  "version": "0.0.7",
+  "name": "LoC Sample Application",
```

Comment:
I don't think that the name of the app can have any spaces - in terminal there's an npm warning that the name is invalid.